Source/Result.swift

```diff
@@ -34,6 +34,29 @@ public enum Result<Value> {
     case Success(Value)
     case Failure(NSData?, NSError)
 
+    /**
+        Creates a `.Success` result with the given value.
+
+        - parameter value: The value to store in the `.Success`.
+
+        - returns: A new `.Success` result.
+    */
+    public init(value: Value) {
```

Comment:
I'm wondering, do they need to be public? Should anyone outside of Alamofire be creating a Result?